wrappers/winuser.go

```diff
@@ -105,26 +106,117 @@ const (
+	SM_CYDOUBLECLK                 = 37
+	SM_CXICONSPACING               = 38
+	SM_CYICONSPACING               = 39
+	SM_MENUDROPALIGNMENT           = 40
+	SM_PENWINDOWS                  = 41
+	SM_DBCSENABLED                 = 42
+	SM_CMOUSEBUTTONS               = 43
+	SM_CXFIXEDFRAME                = SM_CXDLGFRAME
+	SM_CYFIXEDFRAME                = SM_CYDLGFRAME
+	SM_CXSIZEFRAME                 = SM_CXFRAME
+	SM_CYSIZEFRAME                 = SM_CYFRAME
+	SM_SECURE                      = 44
+	SM_CXEDGE                      = 45
+	SM_CYEDGE                      = 46
+	SM_CXMINSPACING                = 47
+	SM_CYMINSPACING                = 48
+	SM_CXSMICON                    = 49
+	SM_CYSMICON                    = 50
+	SM_CYSMCAPTION                 = 51
+	SM_CXSMSIZE                    = 52
+	SM_CYSMSIZE                    = 53
+	SM_CXMENUSIZE                  = 54
+	SM_CYMENUSIZE                  = 55
+	SM_ARRANGE                     = 56
+	SM_CXMINIMIZED                 = 57
+	SM_CYMINIMIZED                 = 58
+	SM_CXMAXTRACK                  = 59
+	SM_CYMAXTRACK                  = 60
+	SM_CXMAXIMIZED                 = 61
+	SM_CYMAXIMIZED                 = 62
+	SM_NETWORK                     = 63
+	SM_CLEANBOOT                   = 67
+	SM_CXDRAG                      = 68
+	SM_CYDRAG                      = 69
+	SM_SHOWSOUNDS                  = 70
+	SM_CXMENUCHECK                 = 71
+	SM_CYMENUCHECK                 = 72
+	SM_SLOWMACHINE                 = 73
+	SM_MIDEASTENABLED              = 74
+	SM_MOUSEWHEELPRESENT           = 75
+	SM_XVIRTUALSCREEN              = 76
+	SM_YVIRTUALSCREEN              = 77
+	SM_CXVIRTUALSCREEN             = 78
+	SM_CYVIRTUALSCREEN             = 79
+	SM_CMONITORS                   = 80
+	SM_SAMEDISPLAYFORMAT           = 81
+	SM_IMMENABLED                  = 82
+	SM_CXFOCUSBORDER               = 83
+	SM_CYFOCUSBORDER               = 84
+	SM_TABLETPC                    = 86
+	SM_MEDIACENTER                 = 87
+	SM_STARTER                     = 88
+	SM_SERVERR2                    = 89
+	SM_MOUSEHORIZONTALWHEELPRESENT = 91
+	SM_CXPADDEDBORDER              = 92
+	SM_DIGITIZER                   = 94
+	SM_MAXIMUMTOUCHES              = 95
+	SM_REMOTESESSION               = 0x1000
+	SM_SHUTTINGDOWN                = 0x2000
+	SM_REMOTECONTROL               = 0x2001
+	SM_CARETBLINKINGENABLED        = 0x2002
+)
+
 var (
 	moduser32 = syscall.NewLazyDLL("user32.dll")
 
-	procCloseDesktop     = moduser32.NewProc("CloseDesktop")
-	procOpenInputDesktop = moduser32.NewProc("OpenInputDesktop")
-	procSetThreadDesktop = moduser32.NewProc("SetThreadDesktop")
+	procCloseDesktop       = moduser32.NewProc("CloseDesktop")
+	procEnumDisplayDevices = moduser32.NewProc("EnumDisplayDevicesW")
+	procGetSystemMetrics   = moduser32.NewProc("GetSystemMetrics")
+	procOpenInputDesktop   = moduser32.NewProc("OpenInputDesktop")
+	procSetThreadDesktop   = moduser32.NewProc("SetThreadDesktop")
 )
 
-func CloseDesktop(desktop syscall.Handle) error {
```

Comment:
Functions should be in alphabetical order.  It's just the constants and structures for which we follow Microsoft's order.